docs/contribute.rst

```diff
@@ -176,11 +180,41 @@ codebase, so sensitive values should not be added there, even temporarily.*
 
 If you need to add a secret value, this currently needs to be added at the GCP
 level by someone with appropriate permissions to edit and apply the Terraform
-configuration. Currently Web-SRE and the backend team have appropriate GCP
+configuration, and to edit the trigger YAML spec to pass through the new secret.
+Currently Web-SRE and the backend team have appropriate GCP
 access and adding a secret is relatively quick. (We can make this easier in
 the future if there's sufficient need, of course.)
 
-**DEPRECATED: Heroku Demo Servers**
+.. note::
+
+  **Always-on vs auto-sleep demo servers**
+
+  The demo servers are on GCP Cloud Run, and by default they will be turned off if
+  there is no traffic for 15 minutes. After this time, the demo app will be woken
+  up if it receives a request.
+
+  Normally, a 'cold start' will not be a problem. However, if the branch you are
+  demoing does things that alter the database (i.e contains migrations), then you
+  may find the restarted demo app crashes because the new migrations have not been
+  applied after a cold start.
+
```

Comment:
Question (non-blocking): any reason we can't set demo1 through to demo5 to be always on? Seems like this could help avoid people forgetting about this behaviour. We often use these demo servers for stakeholder review, so ensuring they are always on might help smooth out potential confusion?

Reply:
Fair question! By default Cloud Run is 'sleepy' so that we only get billed for actual requests handled/CPU used, whereas keeping them always-on has a cost impact. Whether that's a signficant enough upside is TBC, but I think that for most demo uses, which are usually focused on stakeholder signoff of FE work, sleepy apps hopefully won't be a problem - wakeup is about 4-5 secs in my experience. Does that occasional cold-start lag sound OK to you, compared to the Heroku experience?

Reply:
Makes sense! I think the wake up time is fine, I was just meaning to avoid potential server restart errors, but it sounds like those should be relatively uncommon 👍

Reply:
Yeah, i think we'll only have the restart issues if it's a branch that contains migrations, and those are probably 90% done by backendy devs, who already have the access to make a demo service always-on